tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">&nbsp; &nbsp; x = np.zeros(n)
&nbsp; &nbsp; for i in range(n):
&nbsp; &nbsp; &nbsp; &nbsp; x[i] = k % 2
&nbsp; &nbsp; &nbsp; &nbsp; k &gt;&gt;= 1
&nbsp; &nbsp; return x
</pre><p>This code does basically the same as the previously used <code>bitfield</code> but with the output reversed (because of how Qiskit's qubit ordering works). I believe it's better for the users' understanding to stick to one way of doing this in the tutorial, like changing the above code for</p><p> </p><pre class="ql-syntax" spellcheck="false">x = np.binary_repr(k, n) 
x = [x[n-1-i] for i in range(n)]
</pre><p> </p><p>or, if <code>bitfield</code> wasn't inside the <code>brute_force</code> function:</p><p> </p><pre class="ql-syntax" spellcheck="false">x = bitfield(k, n)
x.reverse()
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>